docs/reference/filebeat/configuration-autodiscover.md

```diff
@@ -325,7 +341,11 @@ filebeat.autodiscover:
             equals:
               kubernetes.namespace: kube-system
           config:
-            - type: container
+            - type: filestream
+              id: container-${data.docker.container.id}
+              prospector.scanner.symlinks: true
+			  parsers:
+			    - container: ~
```

Comment:
Again the mix of tabs and spaces. `M-x untabify` should fix this ;)

Also the the provider is Kubernetes, so the `id` needs updateing.
```suggestion
            - type: filestream
              id: container-${data.kubernetes.container.id}
              prospector.scanner.symlinks: true
              parsers:
                - container: ~
```